src/engraving/libmscore/score.cpp

```diff
@@ -486,7 +486,7 @@ void Score::addMeasure(MeasureBase* m, MeasureBase* pos)
 }
 
 //---------------------------------------------------------
-//    fixTicks
+//    setUpTempoMap
```

Comment:
might I suggest `generateTempoMap` or `refreshTempoMap` instead of setUp (which feels weird English)